src/engine/enginedelay.cpp

```diff
@@ -24,12 +24,12 @@
 const int kiMaxDelay = 40000; // 208 ms @ 96 kb/s
```

Comment:
Hmm, good catch. I'm not sure what is the best way to deal with this. It is related to the issue I was dealing with yesterday in #1302 and #1254 with the Echo effect buffers allocating a ridiculous amount of memory. I think we should refactor the engine to allocate buffers for the buffer size actually in use instead of the fixed MAX_BUFFER_SIZE constant and reallocate the buffers when setting a new buffer size from DlgPrefSound. There would need to be a way for EngineDelays and EffectProcessors to reallocate their buffers too. But I do not think this PR needs to take on that issue. I think we should merge this and #1254 first before taking up that issue. As a quick hack we can just increase this constant for EngineDelay.